servers/v3-proxy-api/src/routes/validations/GetSchema.ts

```diff
@@ -108,11 +108,6 @@ export const V3GetSchema: Schema = {
   },
   since: {
     optional: true,
-    isInt: {
-      options: {
-        min: 0,
```

Comment:
removing isInt makes sense, but can you please add a validator method to check it's not < 0 (otherwise may throw an unhandled serialization error)